### ISSUE TITLE
LuCI without OpenWRT

### ISSUE THREAD

**Issue by sampaio23** [openwrt/luci#4909]
Hello!

As far as I'm concerned, I can only install LuCI to a working state if I install using the OpenWRT buildroot environment.
Is there a way to install LuCI without OpenWRT? For instance, if I want to run LuCI in my x86 PC, can I do this? Of course the modules for network configuration would not be available, but I want to create my own modules to execute scripts in my PC.

Thanks in advance! Best regards!

**Comment by hnyman** [comment]
> Is there a way to install LuCI without OpenWRT? 

Not really, as the whole structure is based on being tied to OpenWrt settings.

In theory, perhaps yes, but you would need to re-write pretty much everything.
